.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,60 @@
+---
+name: Bug report (反馈 Bug)
+about: Create a bug report to help HugeGraph improve
+title: '[Bug] Briefly describe the main problem here'
+labels: 'bug'
+assignees: ''
+
+---
+
+<details>
+<summary> Note ( 特别注意, 点击展开 ) </summary>
+
+### Note ( 特别注意 ) : 
+
+> 1. 请先**搜索**, 并**确认**现有的 [Issues](https://github.com/hugegraph/hugegraph/issues) 与 [FAQ](https://hugegraph.github.io/hugegraph-doc/guides/faq.html) 中没有与您相同 / 相关的问题, 请勿重复提交
```

Comment:
tools issues link

Reply:
notice it, consider server-issue-page include many issues of outside module (like `loader/tool/hubble/client`)

(before we transfer all old issues properly) maybe users need search both of them, so I keep the url for server now

(or any suggestion?)

Reply:
prefer to add both server-issue-page and tools-issue-page now.

Reply:
replace FAQ by current issue link now (so as others modules)